plugins/plugins_openapi.yaml

```diff
@@ -78,7 +78,7 @@ paths:
         - in: path
           name: functionName
           schema:
-            type: string
+            type: string9
```

Comment:
Probably not intentional

Reply:
Yes, this makes sense. This was purely a semantics thing, but if it will impact the frontend then it's not worth it. And yes typo there.